deployments/karnot/deployment-config.yaml

```diff
@@ -50,3 +50,29 @@ environments:
           MAX_FELTS_IN_CALLDATA: "22500"
           MONGO_REPLACE_DATA_INSIDE_TRANSACTION: "true"
           STARTING_BLOCK: "0"
+
+  - project: "mainnet"
+    id: 81339901-5926-428c-8e70-3f89a53d3b9d
+    repos:
+      kakarot-rpc:
+        image: ghcr.io/kkrt-labs/kakarot-rpc/node:v0.7.5
+        config:
+          KAKAROT_ADDRESS: "0x15626fc5070f7675966aac4f3a4dc5a9c721cd8df311c6d05611912e3c51c91"
+          UNINITIALIZED_ACCOUNT_CLASS_HASH: "0x7b2de5e73ff7eb338d76c967dd5aa3f3004574d326b8c1402bb819d4983b8b6"
+          RELAYERS_ADDRESSES: "0x219c1e7585b134f94df1a7440676e1f64668cef6ae46cebacf8e12fdb7d1ed7,0x1bc0a910803c09dc59d87d3e414c179c0fb70fca556443b694a33e06f1cb836,0x4756b97b36ff56781960e37f9725e91d4e7f10eabd538a963a16cd718a5b2ac,0x3ad2e0550210d3905881c5eb02b2dc58a4bf9145b286b020e889b76e5d15cc9,0x6f8e544a82e59227712682988d4bbba180f919afae5395bb27c088cba31aac3,0x61e109bf9e9db553c12abfe4b1d4974b71df9b5f999f80731ceb9a8d397b0eb,0x4a0ee234a5a11c942a235b34a2ef09edd61a3ba5803a64455037c430647c901,0x28e72aea13153bad9dcc71be7ad3f4e7bf769fbcbc59d39cd991783c06f126d,0x5407d8d28c43c10bdb7193b2c1553ab881ca77eb2ebc24c1ab02e848455dd99,0x62daa696aa71e97169763fb92729e2ce37a9794da4076e005acf82de6b02449,0x7116732a2ca8f5423581de26a8b316b95f2fa3043e5ac969d556dc511eaf09f,0x72f5988d56d43ddff00e59bdd8298f4c429131b5362e96ed8a12262a3a1d119,0x3c68d3a57ed0e256bc8373318b92c3764d3a460580af0ce66668f0a3f5e0ffc,0x1ed83bb61c70e198ab2a835a9a7657fb7c2c5dded4e9b4de83e4f106e0d07be,0xfd54568b763ffc311f2d058d92053dd707ca8f6d7eadefed9e3f0ad689059c,0x8d57e878e1f12b875d7b4008c15e3fed3323864d7c77966a50f6fb71496314,0x79ebdd91403cea8fc4fb7c53d7a69caea33c2fc0f171a871f8bc8ed38b73a29,0x55fe9ac4db519377c1e1d6331e6a45e3a5ce75bfae7447677be6828c2130317,0x5e5a9efc076bad92061124207e9176b36740c2539c40089fde11e4d07cc90d0,0x43a8ce7a0cb8b714e1c74d19f7b5b0a43f534edc1f7dd3e455fe7eacfa2b181,0x569c7a1ff7e5ac7e18b991ce0b78a97ee7da4710367a293625d4102be782e29,0x58a5b0e3da6eaea2860c200b6977fb08e06a051f2bbc9145d84727f04fe1ab5,0x5746bdc6296ba15a9664afa01062b9ad06f1ff31ac3b70b92eb6f875e618fcb,0x86bb9ec8022d3cc567d07ea2123778a045468c63d87dfdc0c568d20648c5f5,0x30be9c4b8032950f72f14c1b3b77121572be042ddecdf73d97e50682158777a,0x4a24cf6a552591dde0f654c15c42227dbdb66d7634aee144703c61d1b519268,0x595e3ee91ac131371399044b603b5909f33d44bdf72748e5233f6394098f5ba,0xe76bd52863c4d15f8fc4b1e3c51f1ed6c0a29254f5912b2dcfab8752d32f11,0x4806466971baa2a234e49409340ee1c0a703a7865176c01dea458ae0d6a8b06,0x3a427aca2f11e22ef583e131013a81a37054df72ee48dded88c3f1a83463c16"
```

Comment:
Todo: @eugypalu to add mainnet relayers

Reply:
I don’t have permission to push directly to the branch, but here are the relayers:

`RELAYERS_ADDRESSES: "0x436b239cf4ef18010c8dcc41619af73c0a228700b4e91f56a1b19b20038cbca, 0x2d6515424e2f52b740e05a6c2a6620ae70986db0c1877a6cdbffe9afec60751, 0x1c6c8c1192eafcdbca4f49e965ed646dfad87baa84775c88cd3eaaec4723659, 0x548ca41c0ce2faf891acf5248d6bb4037d235ef4e911a5b45a9e6cb0c705ac9, 0x127d6d9f9876acfb9027024782a2fa3ab2facb56faf400a7661cb63c88d6947, 0x4cd32735564d81de5b6de4b6fcf2d29d40073ee386500ed349bcb8e29a6434e, 0x53b694ac311ef28ec82f94367f7795a4fedf30f6915d1e986668af11c4e0a4c, 0x1e84ca7db73489211231b7f92dcf0eeb65536e845d1ce190705af4e5f57e4bb, 0x577f459e1ade342c4ba75ddab31edf3f3f33e450b04499e377a48f704e2fc63, 0x77eebcee4e8f15dff79c4f4c7945f1797024866ff013ca06fe8f2252826fe11, 0x5890db9d10460195b976acf32157786a016f5f07800a44af2b2685b61403944, 0x5f579c4831adc2fba8bf937d5fe71ccd54f2e7c9822340c9116bd21d751d185, 0xb9df2fbb5eeb37b13fcbc19d6003359abfe78cccd2c17d6b9fb2219f34132d, 0x5f7e2276c6f5abc921f5bbaa45a1ecfd6b8a6b151da5b3710d9a07f133217cd, 0x5a35981e4146c3e45e06093058de97db5b3cf5ef6d2a448acb9aea212a94fa7, 0xbb8d3a06dde0751ef04c35871ed84f2f70725d236863f02fa6052520a5c726, 0x6c6504f361ece88619025569f65f8ce49c26e27ffdb4793a9ac91d24a83f343, 0x7ebe2ec1d6a56cae7ea2ef1b7261ea2b3325ad7f28a0731f82ffaa90746d100, 0x407d05d5d511fd7adbdc6f663ab4d28191db4a7780f37eaaa9aeb5a89d76330, 0x1a9a39eda95f35dc35c2c0c79b5feb9d6311c5cde037a4fa9d4ee35a2a5e2e2, 0x5f733bdbf0566eb221a826c7b8791a7465ccefee9b161c5fa223583b76c9739, 0x3dcdca57b0c95c307c8e217048bcb117154b57cbf70e22a0995c64f077e0c2d, 0x32b15b613b2e49c06dc0b7d4a72680976b6eaf9508f0f6d32dcd1ad0ef61470, 0x5d2b80b8bd1e8291b1628801f86c55cf388338a4e9c5100ad2a0d01cb385ad4, 0x31936a338ea6c27730c5ade7beb60942d609850b824a40f0459179034199f79, 0x6488404b7ae3636eb72613b1ce9e28c3d6aa361fea52552f428b7419dc4fa09, 0x6f32af32aa590e63542f3cd9eefbe13d8b4342d2a208a51548235f81dd0db3b, 0x1ff60059dfdbcaa25901136352c1ed4e0b3f6d261fba8bda83628532fd63e9, 0x64c5e7af1ef8811dc3f7c5635682f0aa4d07c2e5a74684e31e1026f8706b390, 0x4e72f0bea2750dd9b54307be0a569aa152153263fa50a267b1bbe5c80974897"`

Reply:
Updated